pkgs/tools/misc/farge/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bash
+, bc
+, feh
+, grim
+, imagemagick
+, slurp
+, wl-clipboard
+, xcolor
+, waylandSupport ? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "farge";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "sdushantha";
+    repo = "farge";
+    rev = "v${version}";
+    hash = "sha256-rTY4vQkrcyVRfIHt1qtFpxaC5bmpJ2/C6NVhEDkxef8=";
+  };
+
+  buildInputs =
+    if waylandSupport
+      then [
+        bash
+        bc
+        feh
+        grim
+        slurp
+        wl-clipboard
+      ]
+      else [
+        bash
+        bc
+        feh
+        xcolor
+      ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    chmod +x farge
+    cp farge $out/bin
```

Comment:
Optional: consider using `install`. It would replace `chmod` and `cp` all at once.

Reply:
Sure, thanks